src/network/networkd-link.c

```diff
@@ -1511,6 +1511,15 @@ static int link_up(Link *link) {
         }
 
         if (link->network->mtu) {
+                /* IPv6 protocol requires a minimum MTU of IPV6_MTU_MIN(1280) bytes
+                   on the interface. Bump up MTU bytes to IPV6_MTU_MIN. */
+                if (socket_ipv6_is_supported() && link->network->mtu < IPV6_MIN_MTU) {
```

Comment:
@poettering  this is per interface basis for example /proc/sys/net/ipv6/conf/enp0s25/disable_ipv6.

Could you please elaborate how we  develope kill switch ?